app/src/main/java/com/example/calculator/InputFragment.kt

```diff
@@ -0,0 +1,190 @@
+package com.example.calculator
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import android.widget.Toast
+import kotlinx.android.synthetic.main.fragment_input.*
+import java.lang.Exception
+
+class InputFragment : Fragment(), View.OnClickListener {
+    private val charArray: Array<String> = arrayOf("+", "-", "*", "/")
```

Comment:
charArray, tên trừu tượng thế nhỉ? 
```suggestion
    private val charArray= arrayOf("+", "-", "*", "/")
```